terraform/modules/rds_migration/role_task_execution.tf

```diff
@@ -0,0 +1,26 @@
+resource "aws_iam_policy" "secret_access" {
+  name = "${var.prefix}-db-migration-secret-access"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = [
+          "secretsmanager:GetSecretValue"
+        ]
+        Effect = "Allow"
+        Resource = [
+          aws_secretsmanager_secret.cloudfoundry_username.arn,
+          aws_secretsmanager_secret.cloudfoundry_password.arn,
+          aws_secretsmanager_secret.cloudfoundry_space.arn,
+          aws_secretsmanager_secret.cloudfoundry_service.arn
+        ]
+      }
+    ]
+  })
+}
+
+resource "aws_iam_role_policy_attachment" "secret_access" {

```

Comment:
This effectively concatenates the above policy to the policies already existing on task execution role from the `application` module